### ISSUE TITLE
Feature Request: respect .lazy modifier on v-model

### ISSUE THREAD

**Issue by jezmck** [shentao/vue-multiselect#1414]
### Steps to reproduce
Add `.lazy` to `v-model` attribute.

### Expected behaviour
Debounced call of `@search-change`.

### Actual behaviour
Immediate calls.

(previously requested: #1194)

**Comment by mariusa** [comment]
Are there any maintainers on this project who could please merge PRs and make new releases? eg the PR above which adds `.lazy`
@shentao @jonascarlbaum @akki-jat @leeniu @arthurshlain

**Comment by jonascarlbaum** [comment]
> Are there any maintainers on this project who could please merge PRs and make new releases? eg the PR above which adds `.lazy`
> @shentao @jonascarlbaum @akki-jat @leeniu @arthurshlain

Am I a maintainer? How do I see that?
I did a PR two years ago and I think it made it into the master branch… But being a maintainer wasn’t something I was aware of or remembers becoming… 😂

**Comment by mariusa** [comment]
I looked at commits history, thought people who committed had the grants to do so...  So it's only @shentao ? Damian, are you all right? Would you consider adding more people as maintainers? (merge PRs, make new releases)

**Comment by shentao** [comment]
I would totally be open to add more maintainers.  Would you be interested?

**Comment by mariusa** [comment]
Yes. Also, people who made contributions in the past, @jonascarlbaum @akki-jat @leeniu @arthurshlain , are you interested?

**Comment by akki-jat** [comment]
@jezmck By default, v-model syncs the input with the data after each input event. We can add the lazy modifier to instead sync after change event. Is this the behavior we want, right?

**Comment by jezmck** [comment]
Sorry, I'm not using Vue in my current job.
